tests/code_fellows/test_linked_list.py

```diff
@@ -1,21 +1,45 @@
 # https://codefellows.github.io/sea-python-401d6/assignments/linked_list.html
 
+import pytest
+
 from code_fellows.linked_list import LinkedList
 
 
 def test_linked_list():
-    assert LinkedList(["head"]).head.value == "head"
-    assert (
-        LinkedList([3, "three"]).head.value == "head"
-        and LinkedList([3, "three"]).head.next.value == 3
-    )
+    assert LinkedList([3, "three"]).head.value == "three"
+    assert LinkedList([3, "three"]).head.next.value == 3
 
 
 def test_Linked_List_push():
     ll = LinkedList([3, "three"])
     assert ll.head.value == "three"
     assert ll.head.next.value == 3
-    ll.push("new_head")
+    ll.push(["new_head"])
```

Comment:
ll.push("new_head") end up acting like this:
ll.push("n")
ll.push("e")
ll.push("w")
ll.push("_")
ll.push("h")
ll.push("e")
ll.push("a")
ll.push("d")

So that ll.head.value == 'd'

I see that is due to:
```
    def push(self, values):
        for value in values:
            self.head = Node(value, next=self.head)
```
I was trying to be future oriented with the for loop - thinking about throwing several values at it at once. But I see that is going to require a bit more nuance than just the simple for loop I made. 

I changed the push function to just do what was asked (haha - you know, the actual assignment ;-) but I am still going to keep it in the follow-up because after all of the initial functions are testing properly, it would be beneficial to modify `push()` to handle something like `push("this", 1, "that")` or `ll.push(["new_head", 3, "test"])` and have the function not return ll.head.value as `["new_head", 3, "test"]` or any of the other odd things you could throw at it that might act differently than expected.